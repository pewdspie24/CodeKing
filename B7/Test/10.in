12
26 569131268
981551 3523830
1322340 881496
9840648 2375328
1073084 6104110
8782866 9827269
1423996 1388872
3786938 9908895
7826390 1917641
1172634 505198
4268086 3420124
4845096 861538
4661572 3184462
512772 2468365
4986851 5911547
6592199 6075876
5354983 3962750
8779760 5740313
9931473 1013851
3377473 9362569
4781297 6426334
417604 3734619
1055662 5210586
1966413 8852442
3354340 6331930
4683437 9115836
8239817 1810746
10 437707561
6761660 7802348
804643 4428862
637372 3791732
3463088 7287866
860140 8681439
423586 8482035
4551052 2933838
3336141 1151603
7937038 3247043
2319429 4715649
30 554957557
7382133 7623992
1011263 4231413
9382094 6140720
4272786 9125459
3939175 3703548
1766200 2816501
6924179 9435779
9909264 6766683
9853598 3609376
2146593 1293877
6934092 3294865
1480764 1906928
5455447 2618643
1861055 6080551
1097305 2763817
8204739 7252169
3225823 2773336
5581182 5060433
7446388 7310105
687842 9077484
344007 2104900
1503698 1114270
7997109 4325780
4296103 5472683
1614105 8161586
7256486 2568688
2444552 1063619
8966062 867193
9883044 2945785
2927644 9041606
15 788762506
808463 4059904
2388372 6223834
6655712 604724
5058202 1121242
387086 4445283
305193 8268212
9105740 5491163
1269399 1061781
3047967 4029608
125821 1751934
2286388 4662477
235175 7770182
2825838 3164734
6483917 3540579
781480 2772340
16 243983660
495965 8742420
5526137 3289593
5947466 7856938
5128876 6993948
1938153 9845606
1302683 8111718
5541021 8333745
9160027 2469328
3055232 1377021
8645227 7480051
9223673 8536285
1382799 323479
8093432 4788578
3848097 1207502
2175396 8437061
6120947 275470
30 199591936
2978191 1355212
9645681 4210253
1663247 7080182
5073455 5457084
9438131 6199612
7651570 6853803
3892584 4597158
8958852 7743077
8484291 1541760
548691 3871618
3210308 1821177
3539965 4770839
6972223 8810789
650446 3036157
7142355 9088032
129646 634213
1213558 4610657
3075757 5322784
4168025 5668784
4244580 5676102
6215671 7447935
124701 6657510
5445199 6429755
7584075 5852079
6773484 1187977
1258667 3675139
402916 4295144
1229430 6874931
587149 9494200
5111260 6104892
10 549668090
8668926 9660552
1087292 8970047
7709074 7544977
9775639 3951416
3795428 9577365
9233341 4625726
7215316 5022604
8044323 8674966
2931089 8813812
7002578 4823720
18 337352158
3819596 8553783
1899875 285280
188925 2927667
4728411 6198063
1848266 4838057
7460898 1653204
7761761 4883321
9624990 5892608
7366946 1863991
7365554 6114572
7031015 5146808
3319208 1691720
8752375 5419902
5625932 1782152
5113563 4603269
1924540 6708181
2727232 7776748
975551 3226155
23 892457828
7667194 4094592
5520340 4958070
155715 7126470
936166 6470960
6024687 9876052
3933774 8728543
4192511 1374047
6478350 4617532
1655343 5014642
6487432 5546591
3750097 9753331
4838364 3382683
1248657 194999
5367039 4536855
4942483 8509021
7932445 2102192
2437281 166299
8262875 2316856
4785999 2255188
3645141 7402461
9213402 2567744
5200274 2925886
2876962 4103335
27 329673812
8458721 382643
6507017 5337752
7275074 3707409
4435120 1168931
2502798 5895396
6901477 9673397
7513925 1188839
3527306 4958118
6202619 7792810
7735354 4991621
7786503 1385679
8707463 8458231
9101424 1657451
6733886 336777
9362973 5406415
334070 541072
7979269 6881760
9031254 1566541
2441476 1403184
5989539 416308
3788076 1056849
2368617 4339199
3721225 8525285
674719 8408921
2354505 1027588
9393204 3984220
8430893 9117157
18 807088934
7381062 9110337
3560747 6565545
7087486 1582622
8019840 8832382
3841291 5454587
9185598 6525826
9437059 5412995
1628734 6342088
6200356 3596190
6219264 7216870
3946836 2156905
1792940 9874658
3369043 889792
6841273 9564455
5038304 6368161
9750043 5899949
6483772 6194617
5012903 5496267
20 181872031
4827963 9360856
7033961 4542377
9975427 8749180
1792728 5932021
9459230 2190630
9645624 5907577
1815972 2409797
9513667 7891367
1942638 3174475
5007368 2271763
3544578 9156389
554232 4816395
3440553 7372525
6294368 7150113
3556534 3696701
7916310 119751
274606 6847989
7468119 6165719
6607912 4420570
8354339 6469880
