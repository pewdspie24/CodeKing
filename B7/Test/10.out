64
93
69
130
37
29
94
58
134
41
146
24
